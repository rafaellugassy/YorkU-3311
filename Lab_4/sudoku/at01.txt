-- Error: game has yet been started
put_number(2, 1, 1)