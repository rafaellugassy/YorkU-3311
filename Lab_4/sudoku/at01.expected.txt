  Setting up a new grid
  0   0   0   0
  0   0   0   0
  0   0   0   0
  0   0   0   0
->put_number(2,1,1)
  Error: Game not yet started
  0   0   0   0
  0   0   0   0
  0   0   0   0
  0   0   0   0
